android/app/src/main/AndroidManifest.xml

```diff
@@ -10,90 +10,98 @@
 
     <application
         android:name=".OpenEventApp"
-        android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+    android:allowBackup="true"
```

Comment:
There's an alignment error, can you pls properly align it, it's a silly one and easy to fix, so pls fix it :)